src/app/Shared/Services/CommandChannel.service.tsx

```diff
@@ -40,12 +40,21 @@ export class CommandChannel {
           if ((jsonResp.dashboardAvailable == true) && (jsonResp.datasourceAvailable == true)) {
             return forkJoin([getDatasourceURL, getDashboardURL])
           } else {
-              return throwError(new Error('ULRs unavailable'));
+            var errMessage = "";
```

Comment:
`var` should be replaced by `let`: https://www.typescriptlang.org/docs/handbook/variable-declarations.html